openpype/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -52,41 +52,47 @@ def process(self, instance):
         filename = "{name}.abc".format(**instance.data)
         path = os.path.join(parent_dir, filename)
 
-        options = {
-            "step": instance.data.get("step", 1.0),
-            "attr": attrs,
-            "attrPrefix": attr_prefixes,
-            "writeVisibility": True,
-            "writeCreases": True,
-            "writeColorSets": instance.data.get("writeColorSets", False),
-            "writeFaceSets": instance.data.get("writeFaceSets", False),
-            "uvWrite": True,
-            "selection": True,
-            "worldSpace": instance.data.get("worldSpace", True)
-        }
-
+        options = {"selection": True}
+        option_keys = [
+            "eulerFilter",
+            "noNormals",
+            "preRoll",
+            "renderableOnly",
+            "uvWrite",
+            "writeColorSets",
+            "writeFaceSets",
+            "wholeFrameGeo",
+            "worldSpace",
+            "writeVisibility",
+            "writeUVSets",
+            "writeCreases",
+            "dataFormat",
+            "step",
+            "stripNamespaces",
+            "verbose",
+            "preRollStartFrame"
+        ]
+        for key in option_keys:
+            options[key] = instance.data[key]
+        self.log.info(options)
```

Comment:
This should not be an info log, a debug log at best but I'd remove it actually.

Reply:
Done